Cannot use jdbi3-bom with JDK 10

Artifact jdbi3-bom cannot be added to `<dependencyManagement>` section:

```
<dependency>
    <groupId>org.jdbi</groupId>
    <artifactId>jdbi3-bom</artifactId>
    <version>3.2.0</version>
    <type>pom</type>
    <scope>import</scope>
</dependency>
```

The error is:

> [ERROR] FATAL ERROR
> [INFO] ------------------------------------------------------------------------
> [INFO] Error building POM (may not be this project's POM).
> Project ID: org.jdbi:jdbi3-parent
> Reason: Invalid JDK version in profile 'jdk9': Unbounded range: [1.9, for project org.jdbi:jdbi3-parent
> org.apache.maven.reactor.MavenExecutionException: Invalid JDK version in profile 'jdk9': Unbounded range: [1.9, for project org.jdbi:jdbi3-parent

Shouldn't the range in jdbi3-parent be defined as `[1.9,)` instead of `[1.9`?

--- a/pom.xml
+++ b/pom.xml
@@ -632,7 +632,7 @@
         <profile>
             <id>jdk9</id>
             <activation>
-                <jdk>[1.9,</jdk>
+                <jdk>[1.9,)</jdk>
             </activation>
             <build>
                 <pluginManagement>

